docusaurus/docs/api/axios-resources.md

```diff
@@ -629,3 +630,315 @@ const sendTelemetryMessages = async (data1, data2, data3, data4) => {
   return { res1, res2, res3, res4 };
 };
 ```
+
+### AvProxyApi
+
+Endpoint for making outbound proxy calls. For a detailed example see [Creating a Proxy](../recipes/proxy.md)
+
+The "X-Availity-Customer-ID" header is required in the config for all methods. This can be done manually when the method is called as shown in the examples below. Alternatively, if the X-Availity-Customer-ID header needs to be added to all (or most) of your api calls, then it can be passed in axios.defaults.headers.common['X-Availity-Customer-ID'] when initializing axios. For more information, see [Config Defaults](https://axios-http.com/docs/config_defaults)
+
```

Comment:
I would think the customerId usually comes from an organization dropdown, maybe provide an example or link to the org dropdown component showing what field exactly this customerId is

Reply:
Good call, I've added more information.